plugins/ltac2/tac2tactics.mli

```diff
@@ -46,6 +46,9 @@ val change : Pattern.constr_pattern option -> (constr array, constr) Tac2ffi.fun
 val rewrite :
   evars_flag -> rewriting list -> clause -> unit thunk option -> unit tactic
 
+val setoid_rewrite :
+  bool option -> constr_with_bindings tactic -> occurrences -> Id.t option -> unit tactic
```

Comment:
Should this be 
```suggestion
  orientation -> constr_with_bindings tactic -> occurrences -> Id.t option -> unit tactic
```
?

Reply:
I only added in this PR an `orientation` type in `tac2qexpr.mli` , which is an alias for `bool option CAst.t` (in my understanding: a parsed optional boolean). You can then convert it to a `raw_tacexpr` (in my understanding: the "core" syntax for Ltac2) with the `of_orient` function (in `tac2quote.ml`), which is registered for handling the `orient` syntactic class in Ltac2 notations.

I guess you are suggesting we should have a separate `orientation` type as well in `tac2tactics.mli`, which certainly makes sense for maintanability / readability. I'll try that.

Reply:
Just pushed the suggested change